### PR TITLE
join Quicksight notification_history table to services

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

add service info to the notification_history table. If this works then I'll join in templates as well as join templates and services with the notifications table.

### CODE REVIEW

File: aws/quicksight/dataset_notifications.tf [hunk 1]
```diff
@@ -50,6 +58,14 @@ resource "aws_quicksight_data_set" "notifications" {
...
         type = "DATETIME"
       }
+      input_columns {
+        name = "feedback_type"
+        type = "STRING"
+      }
+      input_columns {
+        name = "feedback_subtype"
+        type = "STRING"
+      }
```
Comment (sastels) [inline aws/quicksight/dataset_notifications.tf:68]: Added these to notification_history so added them here too so I don't forget later

File: aws/quicksight/dataset_notification_history.tf [hunk 2]
```diff
@@ -38,6 +38,14 @@ resource "aws_quicksight_data_set" "notification_history" {
         name = "notification_status"
         type = "STRING"
       }
+      input_columns {
+        name = "queue_name"
+        type = "STRING"
+      }
```
Comment (jimleroyer) [inline aws/quicksight/dataset_notification_history.tf:44]: Hmm the queue_name is available in the notification_history data set when I logged into Quicksights staging account. It was retrieved automatically before I assume?

**Review by jimleroyer — APPROVED** [review]
LGTM!

File: aws/quicksight/dataset_notification_history.tf (hunk 2 above)
Comment (sastels) [reply aws/quicksight/dataset_notification_history.tf:44]: that is interesting... possibly when I was messing around with joins in the console I reimported things 🤔

**Pull request merged**